CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+* Feat: Client Reports (#829)
```

Comment:
Lets add a note about the breaking change as well, See on Android https://github.com/getsentry/sentry-java/pull/2006
Copy and paste is fine